assets/css/modules/_home.scss

```diff
@@ -1,12 +1,30 @@
-.home { padding: 10px 15px 0 30px; margin-bottom: 2em; }
+.home { padding: 10px 30px 0 30px; margin-bottom: 2em; }
 @media (max-width: 768px) {
   .home {
-    padding: 10px 15px 0px 15px;
+    padding: 10px 30px 0px 30px;
   }
 }
 .col-home {
   &:first-of-type { margin-bottom: 35px; }
+  padding-left: 0;
+  padding-right: 0;
 }
+@media (max-width: 992px) {
+  .col-home {
+    &:last-of-type {
+      margin-top: 25px;
```

Comment:
Can you use `.mt-25` class on the element instead? Or is this only necessary on tablet width?